scripts/skills/perks/perk_rf_hybridization.nut

```diff
@@ -1,21 +1,45 @@
 this.perk_rf_hybridization <- ::inherit("scripts/skills/skill", {
 	m = {
+		IsSpent = false,
```

Comment:
If you set this to `true` whenever a combat ends then you probably also want it `true` by default, so this perk is never visible outside of battle.